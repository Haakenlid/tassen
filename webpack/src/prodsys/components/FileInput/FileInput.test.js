import { FileInputButton } from './index'
import renderer from 'react-test-renderer'

describe('FileInputButton', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<FileInputButton />).toJSON()
    expect(tree).toEqual(expect.any(Object))
    expect(tree).toMatchSnapshot()
  })
})
