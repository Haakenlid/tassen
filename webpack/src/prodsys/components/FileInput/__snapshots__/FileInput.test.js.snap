// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FileInputButton renders correctly 1`] = `
<label
  className="FileInputButton"
>
  upload
  <input
    accept=""
    multiple={true}
    onChange={[Function]}
    type="file"
  />
</label>
`;
